package.json

```diff
@@ -237,6 +237,14 @@
     "@fingerprintjs/fingerprintjs": "^3.4.2",
     "@floating-ui/react": "0.26.9",
     "@glideapps/glide-data-grid": "^6.0.0",
+    "@grafana-plugins/grafana-azure-monitor-datasource": "workspace:*",
+    "@grafana-plugins/grafana-pyroscope-datasource": "workspace:*",
+    "@grafana-plugins/grafana-testdata-datasource": "workspace:*",
+    "@grafana-plugins/jaeger": "workspace:*",
+    "@grafana-plugins/parca": "workspace:*",
+    "@grafana-plugins/stackdriver": "workspace:*",
+    "@grafana-plugins/tempo": "workspace:*",
+    "@grafana-plugins/zipkin": "workspace:*",
```

Comment:
Please remove these - since https://github.com/grafana/grafana/pull/81873 these shouldn't exist. We've [updated the doc steps](https://docs.google.com/document/d/1Fh6NeI7mNQwmtzPMhawMnmHypWd_PD-ZJQ-bDseRy1g/edit#heading=h.4cli68q9dq8p) to reflect this. If we've missed something please let us know.